dcrrates/server/config.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2019, The Decred developers
+// See LICENSE for details.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	flags "github.com/btcsuite/go-flags"
+	"github.com/decred/dcrd/dcrutil"
+)
+
+const (
+	defaultLogPath         = "logs"
+	defaultExchangeRefresh = "20m"
+	defaultExchangeExpiry  = "60m"
+	defaultListen          = ":7778"
+	defaultBtcIndex        = "USD"
+)
+
+var (
+	defaultConfigPath = filepath.Join(dcrutil.AppDataDir("dcrdata", false), "dcrrates-server.conf")
```

Comment:
I think it would be more intuitive if this were dcrrates.conf since the binary is dcrrates.